mlir/docs/SPIRVToLLVMDialectConversion.md

```diff
@@ -817,12 +817,14 @@ to LLVM ops. At the moment, SPIR-V module attributes are ignored.
 
 ## `mlir-spirv-cpu-runner`
 
-`mlir-spirv-cpu-runner` allows to execute `gpu` dialect kernel on the CPU via
-SPIR-V to LLVM dialect conversion. Currently, only single-threaded kernel is
+`mlir-spirv-cpu-runner` is the name of a formerly separate tool that allows to
```

Comment:
I would rename it to something like: 'SPIR-V CPU Runner` and omit the old tool in the description entirely -- I think it should be enough to explain how to run things in the flow

Reply:
Yeah that's a good idea. I've done this now, hope the new version looks good.